src/webview-scripts/CallGraphRenderer.ts

```diff
@@ -28,6 +29,11 @@ interface ICallGraphLayoutNodeLink {
     target: CallGraphLayoutNode;
 }
 
+interface State {
```

Comment:
ESLint should have told you this must use a leading `I` letter. That's the naming convention here.